python/mxnet/_numpy_op_doc.py

```diff
@@ -32,24 +33,70 @@ def _np_reshape(a, newshape, order='C'):
         an integer, then the result will be a 1-D array of that length.
         One shape dimension can be -1. In this case, the value is
         inferred from the length of the array and remaining dimensions.
-    order : {'C'}, optional
+    order : {'C', 'F', 'A'}, optional
```

Comment:
We do not support 'F' and 'A' orders, why did you add them to the doc?

Reply:
in the notes ，i have  give a hint